frontend/__tests__/components/cluster-settings.spec.tsx

```diff
@@ -71,9 +71,6 @@ describe('Cluster Settings page', () => {
   it('should render ClusterSettingsPage component', () => {
     expect(wrapper.exists()).toBe(true);
   });
-  it('should render correct Cluster Settings page title', () => {
-    expect(wrapper.contains('Cluster Settings')).toBeTruthy();
-  });
```

Comment:
Why did you remove this test?

Reply:
Hi @jerolimov ,

It is directly rendering ClusterSettingsPage component and it had <title> component with 'Cluster Settings' text but now I removed <Helmet> in ClusterSettingsPage so the wrapper will not have 'Cluster Settings' text.

Reply:
Ok